src/github/orbit-helpers.js

```diff
@@ -243,12 +243,15 @@ export const orbitAPI = {
   ) {
     try {
       const response = await fetch(
-        `${ORBIT_API_ROOT_URL}/${ORBIT_CREDENTIALS.WORKSPACE}/members/${member}/posts?api_key=${ORBIT_CREDENTIALS.API_TOKEN}`,
+        `${ORBIT_API_ROOT_URL}/${ORBIT_CREDENTIALS.WORKSPACE}/members/${member}/activities?api_key=${ORBIT_CREDENTIALS.API_TOKEN}`,
         {
           method: "POST",
           body: JSON.stringify({
```

Comment:
I see you’re creating a custom activity here, but I think what we want (and used to have) is a Content Activity.

The idea there was to allow “bookmarking” as Content comments on repos outside of your organization. For repos in your organization, the “IssueCommentActivity” will be created by the GH integration automatically. I fear that a `issue_comment:created` custom activity might create confusion with the “native” issue comment activity

Reply:
Yes, you're right. I took a guess here.